### PR TITLE
Page numbers corrected in abstracts from destercke21 to zaffalon21

### PR DESCRIPTION
**Pull request by andrescanoutrera**
Page numbers corrected in abstracts from destercke21 to zaffalon21

### CODE REVIEW

**Comment by andrescanoutrera** [comment]
Page numbers corrected in abstracts from destercke21 to zaffalon21

File: _posts/2021-08-18-kreiss21a.md [hunk 1]
```diff
@@ -36,12 +36,9 @@ id: kreiss21a
...
 tex_title: Towards Improving Electoral Forecasting by Including Undecided Voters and
   Interval-valued Prior Knowledge
-firstpage: !binary |-
-  MjAx4oCTMjA5
-lastpage: !binary |-
-  MjAx4oCTMjA5
-page: !binary |-
-  MjAx4oCTMjA5LTIwMeKAkzIwOQ==
+firstpage: 191
+lastpage: 200
```
Comment (lawrennd) [inline _posts/2021-08-18-kreiss21a.md]:
```suggestion
lastpage: 209
```

File: _posts/2021-08-18-kreiss21a.md [hunk 2]
```diff
@@ -36,12 +36,9 @@ id: kreiss21a
 month: 0
 tex_title: Towards Improving Electoral Forecasting by Including Undecided Voters and
   Interval-valued Prior Knowledge
-firstpage: !binary |-
-  MjAx4oCTMjA5
-lastpage: !binary |-
-  MjAx4oCTMjA5
-page: !binary |-
-  MjAx4oCTMjA5LTIwMeKAkzIwOQ==
+firstpage: 191
```
Comment (lawrennd) [inline _posts/2021-08-18-kreiss21a.md]:
```suggestion
firstpage: 201
```

File: _posts/2021-08-18-kreiss21a.md [hunk 3]
```diff
@@ -36,12 +36,9 @@ id: kreiss21a
...
+lastpage: 200
+page: 191-200
```
Comment (lawrennd) [inline _posts/2021-08-18-kreiss21a.md]:
```suggestion
page: 201-209
```

File: _posts/2021-08-18-moral-garci-a21b.md [hunk 4]
```diff
@@ -24,12 +24,9 @@ id: moral-garci-a21b
 month: 0
 tex_title: Basic Probability Assignments Representable via Belief Intervals for Singletons
   in Dempster-Shafer Theory
-firstpage: !binary |-
-  MjI54oCTMjM0
-lastpage: !binary |-
-  MjI54oCTMjM0
-page: !binary |-
-  MjI54oCTMjM0LTIyOeKAkzIzNA==
+firstpage: 220
```
Comment (lawrennd) [inline _posts/2021-08-18-moral-garci-a21b.md]:
```suggestion
firstpage: 229
```

File: _posts/2021-08-18-moral-garci-a21b.md [hunk 5]
```diff
@@ -24,12 +24,9 @@ id: moral-garci-a21b
...
+lastpage: 228
```
Comment (lawrennd) [inline _posts/2021-08-18-moral-garci-a21b.md]:
```suggestion
lastpage: 234
```

File: _posts/2021-08-18-moral-garci-a21b.md [hunk 6]
```diff
@@ -24,12 +24,9 @@ id: moral-garci-a21b
...
+page: 220-228
```
Comment (lawrennd) [inline _posts/2021-08-18-moral-garci-a21b.md]:
```suggestion
page: 229-234
```

**Review by lawrennd — APPROVED** [review]
I've added a couple of fixes, otherwise great!

**Comment by lawrennd** [comment]
Thanks a lot Andrés!

**Pull request merged**